packages/lodestar/src/api/impl/validator/validator.ts

```diff
@@ -182,6 +206,47 @@ export class ValidatorApi implements IValidatorApi {
     };
   }
 
+  /**
+   * `POST /eth/v1/validator/duties/sync/{epoch}`
+   *
+   * Requests the beacon node to provide a set of sync committee duties for a particular epoch.
+   * - Although pubkey can be inferred from the index we return it to keep this call analogous with the one that
+   *   fetches attester duties.
+   * - `sync_committee_index` is the index of the validator in the sync committee. This can be used to infer the
+   *   subnet to which the contribution should be broadcast. Note, there can be multiple per validator.
+   *
+   * https://github.com/ethereum/eth2.0-APIs/pull/134
+   *
+   * @param validatorIndices an array of the validator indices for which to obtain the duties.
+   */
+  async getSyncCommitteeDuties(epoch: number, validatorIndices: ValidatorIndex[]): Promise<altair.SyncDutiesApi> {
+    this.notWhileSyncing();
+
+    if (validatorIndices.length === 0) {
+      throw new ApiError(400, "No validator to get attester duties");
+    }
+
+    // May request for an epoch that's in the future
+    await this.waitForNextClosestEpoch();
```

Comment:
I think we don't need to as we have current sync committee and next sync committee, and each sync period is ~27h

Reply:
It's important because the validator client will request the next duties exactly at the border between periods. If the clock of the beacon node lags behind the beacon node will reject the request because the validator client is requesting the next period too early. Since our node is way more overloaded than the validator client this happens regularly